x-pack/legacy/plugins/ml/public/jobs/new_job_new/pages/components/charts/loading_wrapper/loading_wrapper.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, Fragment } from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiLoadingSpinner } from '@elastic/eui';
+
+interface Props {
+  hasData: boolean;
+  height: string;
+  loading?: boolean;
+}
+
+export const LoadingWrapper: FC<Props> = ({ hasData, loading = false, height, children }) => {
+  const opacity = loading === true ? (hasData === true ? 0.3 : 0) : 1;
+
+  return (
+    <Fragment>
+      <div
+        style={{
```

Comment:
Could this be moved to a SCSS class?

Reply:
possibly yes, there is a git project task for adding scss to cover the few places styling is needed and is currently being done inline.
https://github.com/elastic/kibana/projects/30#card-23885080